.travis.yml

```diff
@@ -8,7 +8,8 @@ addons:
 
 cache:
     directories:
-        - $HOME/.composer/cache/files
+        - $HOME/.cache/pip
+        - $HOME/.composer/cache
```

Comment:
This is not recommended. It might cause some inconsistencies in composer as it does not update the repository information